clang/include/clang/Basic/DiagnosticSemaKinds.td

```diff
@@ -3286,6 +3286,8 @@ def err_invalid_branch_protection_spec : Error<
   "invalid or misplaced branch protection specification '%0'">;
 def warn_unsupported_branch_protection_spec : Warning<
   "unsupported branch protection specification '%0'">, InGroup<BranchProtection>;
+def err_attribute_invalid_atomic_argument : Error<
+  "invalid argument '%0' to atomic attribute; valid options are: 'remote_memory', 'fine_grained_memory', 'ignore_denormal_mode' (optionally prefixed with 'no_')">;
```

Comment:
This should be clang-formatted which would add a word-wrap.  No idea why the bot isn't complaining about it though.

Reply:
will wrap it to 80 characters per line